### PR TITLE
Propose meetup types

### PR DESCRIPTION
**Pull request by danielmocan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,15 @@ Choosing a date for the meetup isn't always a straight-forward decision. Things
 to choose doesn't have another competing meetup group scheduled for it already.
 - Take into account holidays, events, or company activities to make sure none of those conflict with the meetup date.
 
+### Meetup Type
+
+Its very easy to think about meetups in a clasical way: you have a speaker (or more) that comes and delivers a presentation and its nothing wrong with that. Its important to have different types of meetups, so we propose something like this:
```
Comment (lirantal) [inline README.md]:
```suggestion
Meetups in their classic formats feature a speaker (or more) who delivers a presentation. However, meetups are a great ground to also feature different formats of learning. We propose:
```

File: README.md [hunk 2]
```diff
@@ -144,6 +144,15 @@ Choosing a date for the meetup isn't always a straight-forward decision. Things
...
+- **presentation** - Clasical presentations (one or more subjects)
+- **workshop** - Coding during a meetup is very imporant, it helps retain more information because you have to get your hands dirty :)
```
Comment (lirantal) [inline README.md]:
```suggestion
- **workshop** - Writing code during a meetup helps attendees experience and learn through practical hands-on experience.
```

File: README.md [hunk 3]
```diff
@@ -144,6 +144,15 @@ Choosing a date for the meetup isn't always a straight-forward decision. Things
...
+- **open talks** - You propose a topic, then the first thing at the begining of the meetup you ask people what subject releated to that topic they would like to be disscused. For this type of meetup would be nice to have a whiteboard to write the subject on it. They you go through each subject and anyone can share their knowleadge. Its very important that you have some that moderates the coversation, else you get sidetracket very easy.
```
Comment (lirantal) [inline README.md]:
```suggestion
- **open talks** - A topic is selected for the meetup, and during the meetup event the attendees share a collection of subjects related to the topic proposed. A whiteboard helps facilitate this activity. Once enough subjects are gathered an open discussion with attendees can take place to share knowledge. An important aspect of this meetup is having a moderator for the conversation so it doesn't get side-tracked.
```

File: README.md [hunk 4]
```diff
@@ -144,6 +144,15 @@ Choosing a date for the meetup isn't always a straight-forward decision. Things
...
+- social - For this meetup you dont have a specific subject, but its a good way for the members of you comunity to socialize. Most of the times during regular meetups there is little time to socialize.
```
Comment (lirantal) [inline README.md]:
```suggestion
```

Comment (lirantal) [reply README.md]: Since we aren't proposing anything new here I think we can leave this one out.

**Comment by lirantal** [comment]
Thanks @danielmocan, looks good!
I updated with some suggestions to rephrase the texts.

**Review by lirantal — CHANGES_REQUESTED** [review]

**Comment by danielmocan** [comment]
Thank you @lirantal for helping me with the rephrase.

**Review by lirantal — APPROVED** [review]

**Pull request merged**